apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -156,7 +156,20 @@ export default function GlobalStyle () {
                   cursor: pointer;
                 }
               }
-
+              
+              .ant-table-row-expand-icon-spaced {
+                display: none;
+              }
+              
+              .ant-table-row-expand-icon {
+                color: ${colors.green[6]};
+                border-color: ${colors.green[6]};
+                border-radius: 1px;
```

Comment:
css hack detected, seems like it's design mistake, there is no 1px border-radius

Reply:
removed styling, now using antd icon instead